README.md

```diff
@@ -29,7 +29,21 @@ Install tkinter to be able to display graphs
 `sudo apt install python3-tk`
 
 ### Windows  
-Help wanted  
+Make sure pip for python 3 is installed  
+`pip3 --version` should not print an error  
+  
+Install pipenv via pip  
+`pip install pipenv`  
```

Comment:
is pip for python 3 not `pip3` on windows?

Reply:
Well, I think pip operates on the current env context, and pip3 operates only for python3.